server/plugins/task-plugin-api/src/main/java/com/oceanbase/odc/plugin/task/api/datatransfer/dumper/ExportOutput.java

```diff
@@ -66,12 +66,16 @@ public class ExportOutput {
     private final BinaryFile<Manifest> manifest;
     private final BinaryFile<List<?>> checkpoints;
     private final List<DumpDBObject> dumpDbObjects;
+    private boolean isZip;
```

Comment:
no `is` prefix for boolean field, use `zip` straightly

Reply:
OK